packages/uniforms-bridge-graphql/src/GraphQLBridge.ts

```diff
@@ -124,32 +106,26 @@ export default class GraphQLBridge extends Bridge {
       return {};
     }
 
-    const defaultValue = this.getField(name).defaultValue;
-
-    return defaultValue === undefined
-      ? this.extras[name]?.initialValue
-      : defaultValue;
+    const { defaultValue } = this.getField(name);
+    return defaultValue ?? this.extras[name]?.initialValue;
   }
 
-  // eslint-disable-next-line complexity
   getProps(nameNormal: string, props: Record<string, any> = {}) {
     const nameGeneric = nameNormal.replace(/\.\d+/g, '.$');
 
-    const field = this.getField(nameGeneric, false);
-    const fieldType = extractFromNonNull(field).type;
-
+    const { name, type } = this.getField(nameGeneric);
     const ready = {
-      required: graphql.isNonNullType(field.type),
+      required: isNonNullType(type),
       ...this.extras[nameGeneric],
       ...this.extras[nameNormal],
     };
```

Comment:
This object is called ready but it's not ready yet. We can prepare this object at once but it may cause few conditions more, or just change its name to more suitable. Maybe Ruby-like `result` would fit here nicely?

Reply:
For you it's Ruby - for me it's Nim :stuck_out_tongue:

Reply:
Yeah, whoever made it up - it's a very nice solution for this use case ;)